### PR TITLE
Prac 05 feedback

### PR DESCRIPTION
**Pull request by JarrodE**
Hey, @JoshuaDelosSantos could you please review my code and tell me if I can better my variable, function, and commit names and any other problems in my code?
Cheers mate!

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+"""
+Emails
+Estimated: 45 minutes
+Actual: 50 minutes
+"""
+
+
+def main():
```
Comment (JoshuaDelosSantos) [inline prac_05/emails.py:8]: Docstrings might be a good idea 😊

File: prac_05/wimbledon.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+def read_wimbledon_data(filename, winners):
+    """Read Wimbledon data from A CSV file and populate winners list."""
+    with open(filename, 'r', encoding="utf-8-sig") as in_file:
+        in_file.readline()
+        for line in in_file:
+            parts = line.strip().split(",")
+            winners.append(parts)
+
+
+def count_wins_and_countries(winners, champion_to_wins, countries):
```
Comment (JoshuaDelosSantos) [inline prac_05/wimbledon.py:30]: Should this function return something?

File: prac_05/word_occurrences.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+def count_words(text):
+    """Converts words to lowercase and counts them."""
+    word_to_count = {}
+    words = text.split()
+    for word in words:
+        word = word.lower()
+        word_to_count[word] = word_to_count.get(word, 0) + 1
+    calculate_longest_word(word_to_count)
```
Comment (JoshuaDelosSantos) [inline prac_05/word_occurrences.py:22]: Should you be calling a function in a function?

**Review by JoshuaDelosSantos — COMMENTED** [review]
Gave a few feedbacks @JarrodE 
Could possibly have more commit messages next time (small incremental commits)
🙂

File: prac_05/word_occurrences.py (hunk 3 above)
Comment (lindsaymarkward) [reply prac_05/word_occurrences.py:22]: @JoshuaDelosSantos - There's nothing wrong with calling a function in a function.
However, this is function chaining, breaching SRP. A function should only call something that does a part of its one job, not that does the next job.
"calculate" should not print.

**Pull request merged**